web/packages/teleport/src/components/FormLogin/FormIdentifierFirst.test.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { http, HttpResponse } from 'msw';
+import { setupServer } from 'msw/node';
+
+import { render, screen, userEvent, waitFor } from 'design/utils/testing';
+
+import { storageService } from 'teleport/services/storageService';
+
+import { FormIdentifierFirst } from './FormIdentifierFirst';
+
+const server = setupServer();
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+test('no user remembered in localstorage renders input form', () => {
+  jest.spyOn(storageService, 'getRememberedSsoUsername').mockReturnValue('');
```

Comment:
Can we just set the value in local storage instead of mocking the storage service?